components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -202,14 +220,146 @@ The member expression dialog also supports tokens, because candidate tokens are
+
+```
+"infra:*",
+"compliance:*",
+"system:*",
+"event:*",
+"ingest:*",
+"secrets:*",
+"telemetry:*",
+"iam:projects:list",
+"iam:projects:get",
+"iam:projects:assign",
+"iam:policies:list",
+"iam:policies:get",
+"iam:policyMembers:*",
+"iam:teams:list",
+"iam:teams:get",
+"iam:teamUsers:*",
+"iam:users:get",
+"iam:users:list"
+```
+
+In order to create a Project Owner, you can use an existing user, or create a user with username `test_project_owner` by following [Creating Users]({{< relref "users.md#creating-users" >}}).
+
+Next, you will create a policy that gives `test_project_owner` the role of `project-owner` for your new project. You may choose the and ID. The policy will look like the JSON below:
+
+```json
+{
+  "name": "Test Project Policy",
+  "id": "test-project-policy",
+  "projects": ["test-project"],
+  "members": [ "user:local:test_project_owner"],
+  "statements": [
+    {
+      "effect": "ALLOW",
+      "role": "project-owner",
+      "projects": ["test-project"]
+    }
+  ]
+}
+```
+
+By adding `test-project` to the policy's top-level `projects` field, we ensure that the Project Owner has access to this policy.
+
+Save as a JSON file and follow the steps in [Creating a Policy]({{< relref "iam-v2-api-reference.md#creating-a-policy" >}}) to pass that data to Chef Automate.
+
+#### Editor and Viewer Policies for Projects
+
+You (or your new Project Owner) may additionally choose to create a `Test-Project Editors` policy and a `Test-Project Viewers`
+policy. Once those are in place, you or your `Test-Project Owner` can easily add members to each policy
+to give them privileges to view or edit resources assigned to the `Test Project` project. To create
+those policies, use the JSON below:
+
+Viewers Policy:
+
+```json
+{
+  "name": "Test Project Viewers",
+  "id": "test-project-viewer-policy",
+  "projects": ["test-project"],
+  "members": [],
+  "statements": [
+    {
+      "effect": "ALLOW",
+      "role": "viewer",
+      "projects": ["test-project"]
+    }
+  ]
+}
+```
+
+Editors Policy:
+
+```json
+{
+  "name": "Test Project Editors",
+  "id": "test-project-editor-policy",
+  "projects": ["test-project"],
+  "members": [],
+  "statements": [
+    {
+      "effect": "ALLOW",
+      "role": "editor",
+      "projects": ["test-project"]
+    }
+  ]
+}
+```
+
+Now, if you would like Terry to be able to view `Test Project` and Kelly to be able to edit `Test Project`, you'll add them
+as members to the relevant policies. You can do that directly in the above JSON when creating the policy, or in the browser
+by selecting **Settings**, then **Policies**, then `Test-Project Policy`. Under the **Details** tab, you will find the
+option to select new policy members.
+
+Assuming Terry is not a member of any other policy, once you make Terry a member of the `Test-Project Viewer` policy,
+they will only be able to see resources assigned to `Test Project`. They will not be able to update or delete them.
```

Comment:
i prefer gender neutral

Reply:
A couple points. "They" is already stretching it; it is for informal writing rather than documentation like this. But even if we let that slide, using "they" to refer to an antecedent that is a proper name really sounds strange! That is, this is passably acceptable: "The participant indicated their preference." But this? "Bob indicated their preference".

Reply:
I don't know if I think these docs are particularly formal-- we use the imperative form pretty consistently, which is traditionally a no.

I definitely think there was a time-- and there are still formal spaces-- where the authors prefer to be gender-indicative (s/he); however, we live in a pretty different space now where the question of s/he is ambiguous in and of itself. This is to say, just because someone has a name we might associate with a "she", that person might not identify as such. So then what do we use? "It"? That's not particularly humanizing.

We do have an opportunity to assign gender here for the sake of maintaining the more traditional pronoun/possessive pairing, but we _also_ have the opportunity --subtly-- to suggest that it's not relevant/that we support all individuals regardless of their gender. 

Here's a little more on that: https://www.mypronouns.org/they-them.

As a woman, I find myself continually frustrated that for a long, long time, most technical resources (arguably, all resources, all the way to kids books) feature a male "representative". The "they" allows us to be inclusive. Happy to chat more in person.

Ultimately, though, this question might be one for the docs team.